specs/packagemanager/PackageManagement.md

```diff
@@ -0,0 +1,1134 @@
+        Boolean AllowUnsigned;
+        Boolean DeveloperMode;
+        Boolean ForceAppShutdown;
+        Boolean ForceTargetAppShutdown;
+        Boolean ForceUpdateFromAnyVersion;
+        Boolean InstallAllResources;
+        Boolean RequiredContentGroupOnly;
+        Boolean RetainFilesOnFailure;
+        Boolean StageInPlace;
+        Boolean DeferRegistrationWhenPackagesAreInUse;
+
+        Boolean IsExpectedDigestsSupported { get; };            // Requires Windows >= 10.0.22621.0 (aka Win11 22H2)
+        IMap<Windows.Foundation.Uri, String> ExpectedDigests{ get; };
+
+        Boolean IsLimitToExistingPackagesSupported { get; };    // Requires Windows >= 10.0.22621.0 (aka Win11 22H2)
+        Boolean LimitToExistingPackages;
+    }
+
+    // Requires Windows >= 10.0.19041.0 (aka 2004 aka 20H1)
+    [contract(PackageDeploymentContract, 1)]
+    runtimeclass StagePackageOptions
+    {
+        StagePackageOptions();
+
+        PackageVolume TargetVolume;
+        IVector<Windows.Foundation.Uri> DependencyPackageUris { get; };
+        IVector<String> OptionalPackageFamilyNames { get; };
+        IVector<Windows.Foundation.Uri> OptionalPackageUris { get; };
+        IVector<Windows.Foundation.Uri> RelatedPackageUris { get; };
+        Windows.Foundation.Uri ExternalLocationUri;
+        Windows.Management.Deployment.StubPackageOption StubPackageOption;
+        Boolean DeveloperMode;
+        Boolean ForceUpdateFromAnyVersion;
+        Boolean InstallAllResources;
+        Boolean RequiredContentGroupOnly;
+        Boolean StageInPlace;
+        Boolean AllowUnsigned;
+
+        Boolean IsExpectedDigestsSupported { get; };            // Requires Windows >= 10.0.22621.0 (aka Win11 22H2)
+        IMap<Windows.Foundation.Uri, String> ExpectedDigests{ get; };
+    }
+
+    // Requires Windows >= 10.0.19041.0 (aka 2004 aka 20H1)
+    [contract(PackageDeploymentContract, 1)]
+    runtimeclass RegisterPackageOptions
+    {
+        RegisterPackageOptions();
+
+        PackageVolume AppDataVolume;
+        IVector<Windows.Foundation.Uri> DependencyPackageUris { get; };
+        IVector<String> OptionalPackageFamilyNames { get; };
+        Windows.Foundation.Uri ExternalLocationUri;
+        Boolean DeveloperMode;
+        Boolean ForceAppShutdown;
+        Boolean ForceTargetAppShutdown;
+        Boolean ForceUpdateFromAnyVersion;
+        Boolean InstallAllResources;
+        Boolean StageInPlace;
+        Boolean AllowUnsigned;
+        Boolean DeferRegistrationWhenPackagesAreInUse;
+
+        Boolean IsExpectedDigestsSupported { get; };            // Requires Windows >= 10.0.22621.0 (aka Win11 22H2)
+        IMap<Windows.Foundation.Uri, String> ExpectedDigests{ get; };
+    }
+
+    // Requires Windows >= 10.0.19041.0 (aka 2004 aka 20H1)
+    [contract(PackageDeploymentContract, 1)]
+    runtimeclass RemovePackageOptions
+    {
+        RemovePackageOptions();
+
+        Boolean OkIfNotFound;
```

Comment:
I assume `OkayIfNotFound` defaults to true.

Maybe `SucceedIfNotFound`? or flip it `PackageMustExist`?